src/main/java/org/prebid/server/handler/ExceptionHandler.java

```diff
@@ -23,7 +23,7 @@ public static ExceptionHandler create(Metrics metrics) {
 
     @Override
     public void handle(Throwable exception) {
-        logger.warn("Error while establishing HTTP connection", exception);
+        logger.warn("Generic error handler: {0}, cause: {1}", exception.getMessage(), exception.getCause());
```

Comment:
Potentially cause may be missing.
Suggest to make separate method to fetch error message, like:
```
String errorMessageFrom(Throwable exception) {
  return exception != null ? exception.getMessage() : "";
}
```
So, you can use it as:
```
logger.warn("Generic error handler: {0}, cause: {1}", 
    errorMessageFrom(exception), errorMessageFrom(exception.getCause()));
```
But up to you.
The idea is to rid off any stacktraces in logs for uncatched errors.